test/test_unit_client.py

```diff
@@ -81,13 +79,13 @@ def test_client_passes_init_parameters_with_none_provided(self):
         Client.initialize(None, None, None, False)
         client.initialize.assert_called_once_with(None, None, None, False)
 
-    @patch('conjur.api.client.ApiConfig', new=MissingMockApiConfig)
-    def test_client_init_initialize_calls_load_properly(self):
-        client = Client
-        mock_init_controller = InitController
-        mock_init_controller.load = MagicMock()
-        Client.initialize('someurl', 'someaccount', '/some/path/to/pem', False, True)
-        mock_init_controller.load.assert_called_once()
+    # @patch('conjur.api.client.ApiConfig', new=MissingMockApiConfig)
```

Comment:
why this commented code

Reply:
because the UX wasnt finalized so didnt want to write UTs. Updating now